README.md

```diff
@@ -1,22 +1,336 @@
-# Diy Json Serializer
+# DIY JSON Serializer
 
 ## Objectives
 
-  1. Understand what JSON is
-  2. Create and Return simple JSON Objects
-  3. Load those objects with AJAX into HTML
-
-## Outline
-
-  * Grab the code from previous readme.
-  * Goal here is to re-do the post show page to be data backed by JS
-  * Previously, we just dumped data out at different routes. This isn't sustainable. We need a way to hit just one route, and get structured data back.
-  * Traditionally this was solved with HTML. HTML has both data and formatting, not good
-  * Just want pure data, used to be given via XML. XML was annoying
-  * Then came JSON
-  * Show them what JSON looks like. Just a dictionary or array of dictionaries!
-  * JSON is a way of "serializing" data. Remember we need to send everything as strings. Serialization means convert from the ruby format of an object, to a string format of an object. 
-  * Let's write our own JSON post serializer.
-  * Make it a separate class with a single class method that takes in the post object and returns a string of JSON
-  * don't use `to_json` or anything like that. Have them by hand build up a JSON dictionary of the important bits of the post
-  * The show page, make it backed by JSON.
+  1. Understand what JSON is.
+  2. Create and Return simple JSON Objects by hand.
+  3. Load those objects with AJAX into HTML.
+
+## Lesson
+
+Continuing with our blog application, we'll be improving our blog post
+API to provide more structure and more efficient data access. Make sure
+to run `rake db:seed` to set up some starting data!
+
+Last time, we created an endpoint for the single piece of data we wanted
+to access. That worked well enough, but doesn't scale. If we want to use
+AJAX to get all the data from a blog post, we'd be creating so many
+endpoints, and making so many GET requests.
+
+What we really need is one endpoint to make one request and get back
+structured data representing our blog post.
+
+### Serialization
+
+If we're directly accessing a post from our web browser, the answer is
+simple. The controller renders a template that contains HTML and the
+data we need.
+
+We could still do this for an API endpoint, but then that HTML would
+probably get in our way. We want the data, not the formatting. This,
+like so many things, comes back to Separation of Concerns. An API
+endpoint is only concerned with returning the data needed; it's up to
+the API consumer, in this case our AJAX call, to decide how to display
+it.
```

Comment:
This distinction is really well put. I dig it.

I'm a little curious about "separation of concerns" -- it's meaning is intuitive, but I wonder if we can emphasize it as a phrase that the student should know by bolding it rather than capitalizing it. The caps somehow make it seem really official and make me feel like I missed something.